php-frontend/src/main/java/org/sonar/plugins/php/api/tests/PHPCheckTest.java

```diff
@@ -19,17 +19,16 @@
  */
 package org.sonar.plugins.php.api.tests;
 
-import com.google.common.base.Charsets;
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableList.Builder;
 import com.sonar.sslr.api.typed.ActionParser;
-import java.io.File;
 import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.nio.charset.StandardCharsets'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)